platform/src/lib/envConstants.js

```diff
@@ -1 +1 @@
-export const LEGACY_PLATFORM_ENV = process.env.NEXT_PUBLIC_ORIGINAL_PLATFORM;
+export const NEXT_PUBLIC_AUTHORISATION = process.env.NEXT_PUBLIC_AUTHORISATION;
```

Comment:
Is there a specific reason we are using authorisation from the runtime environment? This is because this approach isn't right, and gives users access to all information without having been logged it.

Can we create a login feature first to allow users authenticate themselves before we give them access to our data?